Dockerfile

```diff
@@ -18,17 +25,9 @@ COPY pkg/ pkg/
 COPY .git .git
```

Comment:
I think we can drop the `.git` folder, here!

Reply:
@prometherion done it!